test/action.yml

```diff
@@ -52,7 +52,7 @@ runs:
         go test -p 1 -timeout 30m -v ./suites/${{inputs.name}}
 
     # Print snap logs
-    - if: always() && ${{inputs.print_logs}} != 'false'
+    - if: ${{inputs.print_logs}} == 'true'
```

Comment:
`always()` is there so that the step runs even if the tests fail.

Reply:
`${{inputs.print_logs}} == 'true'` is wrong. It should `${{inputs.print_logs == 'true'}}` or `inputs.print_logs == 'true'`